test/terra/backends/qasm_simulator/qasm_snapshot.py

```diff
@@ -206,12 +206,12 @@ def stabilizes_statevector(stabilizer, statevector):
         # Get stabilizer and destabilizers and convert to sets
         for stab in stabilizer:
             if stab[0] == '-':
-                pauli_mat = -1 * Pauli.from_label(stab[1:]).to_matrix()
+                pauli_mat = -1 * Pauli(stab[1:]).to_matrix()
             else:
-                pauli_mat = Pauli.from_label(stab).to_matrix()
-            val = statevector.conj().dot(pauli_mat.dot(statevector))
-            if not np.isclose(val, 1):
-                return False
+                pauli_mat = Pauli(stab).to_matrix()
+        val = statevector.conj().dot(pauli_mat.dot(statevector))
+        if not np.isclose(val, 1):
+            return False
```

Comment:
This change brings some logic outside a loop, which looks like it belongs inside?  `pauli_mat` depends on the loop variable `stab`.

Reply:
Oops, good catch I messed up the tab indents here

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1303/commits/bc9951211434634964784c0fd076782126a94425